src/pages/preferences/preferencesTabs/cliTab/VerifyCard.tsx

```diff
@@ -22,10 +22,8 @@ export const VerifyCard = () => {
 
   const latestRevision = get(data, "clientConfig.latestRevision", "");
   const verificationCode = `
-"[evergreen] ${format(new Date(), "yyyy/MM/dd hh:mm:ss")} [p=notice]:
-[message='Binary is already up to date - not updating.'
-revision='${latestRevision}']"
-  `;
+"[message='Binary is already up to date - not updating.' revision='${latestRevision}']"`;
```

Comment:
Can you also remove the dependency on line 6. `import { format } from "date-fns";`

Reply:
Done